### PR TITLE
Multiple prefixes in core

### PR DESCRIPTION
**Pull request by roeslpa**
Solves #391

### CODE REVIEW

File: src/main/java/de/qabel/core/accounting/AccountingHTTP.java [hunk 1]
```diff
@@ -120,6 +121,35 @@ public void updateProfile() throws IOException, QblInvalidCredentials {
...
+		HttpGet httpGet = new HttpGet(uri);
+		httpGet.addHeader("Authorization", "Token " + server.getAuthToken());
+		try (CloseableHttpResponse response = httpclient.execute(httpGet)) {
+			HttpEntity entity = response.getEntity();
+			if (entity == null) {
+				throw new IOException("No answer from login");
+			}
+			String responseString = EntityUtils.toString(entity);
+			if(responseString.length() > 6) {
+				prefixes = new ArrayList<>(Arrays.asList(
```
Comment (audax) [inline src/main/java/de/qabel/core/accounting/AccountingHTTP.java]: Why don't you parse the JSON with the gson library?

File: src/test/java/de/qabel/core/accounting/AccountingHTTPTest.java [hunk 2]
```diff
@@ -75,7 +75,7 @@ public void testGetCredentials() throws IOException, QblInvalidCredentials {
 
 	@Test
 	public void testGetPrefix() throws IOException, QblInvalidCredentials {
-		assertNotNull(accountingHTTP.getPrefix());
+		assertNotNull(accountingHTTP.getPrefixes());
```
Comment (audax) [inline src/test/java/de/qabel/core/accounting/AccountingHTTPTest.java]: You should also check that the list is not empty to fully substitute the old test.

**Comment by roeslpa** [comment]
Gson is now used, test is adapted and a method to create prefixes is implemented since the user had no prefix which could have been requested.

**Comment by audax** [comment]
ack

**Pull request merged**